group_vars/osds.yml.sample

```diff
@@ -7,14 +7,6 @@
 # file as a good configuration file when no variable in it.
 dummy:
 
-# Variables here are applicable to all host groups NOT roles
-
-# This sample file generated by generate_group_vars_sample.sh
-
-# Dummy variable to avoid error because ansible does not recognize the
-# file as a good configuration file when no variable in it.
-#dummy:
-
```

Comment:
This is a bug that needs to be fixed in `generate_group_vars_sample.sh` instead.
Otherwise, this would be regenerated every time we run it.